src/remove.js

```diff
@@ -0,0 +1,11 @@
+const removeTask = (i) => {
+  let tasks = JSON.parse(localStorage.getItem('datas'));
+  tasks = tasks.filter((task) => task.description !== tasks[i].description);
+  /* eslint-disable no-plusplus */
+  for (let i = 0; i < tasks.length; i++) {
+    tasks[i].index = i;
+  }
```

Comment:
Please, try to avoid using classic loop and use forEach(), instead.

Reply:
Thank you for this feedback